app/code/Magento/QuoteGraphQl/Model/Cart/SetBillingAddressOnCart.php

```diff
@@ -90,7 +90,7 @@ public function execute(ContextInterface $context, CartInterface $cart, array $b
             );
         }
 
-        $billingAddress = $this->createBillingAddress($context, $customerAddressId, $addressInput);
+        $billingAddress = $this->createBillingAddress($context, $customerAddressId, $addressInput,$sameAsShipping);
```

Comment:
Please, make sure you preserve the PSR code style and add whitespace before the argument